tests/e2e/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Do we still need a license?

Reply:
I don't know the legal intricacies, but I don't think you can remove a license?

Reply:
Technically this new package is pure addition in this repo, and it's not even exposed as a public package.

Also, the whole monorepo is MIT so this directory inherits it.